ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -2456,16 +2456,7 @@ periodics:
           - as: launch
             steps:
               cluster_profile: hypershift
-              test:
-              - as: publish-console-url-hack
-                commands: echo "https://cluster.example.com" > "${SHARED_DIR}/console.url"
-                from: cli
-                resources: { requests: { cpu: 100m, memory: 100Mi } }
-              - as: wait
-                commands: sleep $(CLUSTER_DURATION) & wait
-                from: cli
-                resources: { requests: { cpu: 100m, memory: 100Mi } }
-              workflow: hypershift-launch-aws
+              workflow: hypershift-aws-conformance
```

Comment:
At a minimum, you'll need to add the following step to give the cluster-bot access to monitor the ephemeral namespace:
https://steps.ci.openshift.org/reference/openshift-cluster-bot-rbac

If you want anyone to be able to use this cluster after the hypershift-aws-conformance test is run, then you'll also need to add something like this as well to your test:
https://github.com/openshift/release/blob/053c3418167d3c4a18aff90b937a69061a36c25d[…]r/jobs/openshift/release/openshift-release-infra-periodics.yaml